server/models/index.js

```diff
@@ -1,7 +1,38 @@
-import Game from './Game';
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(module.filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(`${__dirname}/../config/config.json`)[env];
+const db = {};
 
+let sequelize;
+if (config.use_env_variable) {
```

Comment:
I'm ok with doing config variables, I find them nicer to work with, and requires less setup when deploying